styles/main.css

```diff
@@ -289,49 +316,52 @@ flex-direction: column ;
 order: 0;
 color: blue;  
 }
-@media screen and(max-width: 725px) {
-#topul {
-  flex-direction: column;
-  align-items: center;
-  padding: 0.5em;
-  border-top: none;
-  text-decoration: underline;
-} 
+}
   
+.sandwich-icon,
+#menu-toggle {
+  display: none;
 }
-.sandwhich-icon {
+
+@media screen and ( max-width: 500px ) {
+.sandwich-icon {
 display: block;
 width: 40px;
 height: 25px;
-border-top: 5px;
+border-top: 5px solid #116530;
 margin: 1em auto;
 transition: 200ms;
 }
-.sandwhich-icon:hover {
+.sandwich-icon:hover {
 cursor: pointer;
 }
-.sandwhich-icon::after
-.sandwhich-icon::before {
+.sandwich-icon::after,
+.sandwich-icon::before {
+content: " ";
 display: block;
 width: 40px;
 border-top: 5px solid #116530;
 position: absolute;
 transition: transform 200ms;
 }
-.sandwhich-icon label {
+.sandwich-icon {
 position: relative;
 }
```

Comment:
The .sandwich-icon element is itself a label, the 'label' selector is not correct here